lib/perl/Genome/Model/Tools/Picard.t

```diff
@@ -61,4 +62,52 @@ for (0..6) {
 
 };
 
-1;
+subtest "Version compare" => sub {
+    my @less_cases = (
+        ["1.85", "1.123"],
+        ["1.05", "1.05.1"]
+        );
+
+    for my $v (@less_cases) {
+        ok($pkg->version_compare($v->[0], $v->[1]) < 0, "$v->[0] < $v->[1]");
+        ok($pkg->version_compare($v->[1], $v->[0]) > 0, "$v->[1] > $v->[0]");
+        ok($pkg->version_compare($v->[0], $v->[0]) == 0, "$v->[0] == $v->[0]");
+        ok($pkg->version_compare($v->[1], $v->[1]) == 0, "$v->[1] == $v->[1]");
+    }
+};
+
+subtest "Enforce minimum version" => sub {
+    # These are ordered from greatest to least
+    my @versions = $pkg->_versions_serial;
```

Comment:
To clarify, the only reason _versions_serial exists is to get at the even indexed elements of the PICARD_VERSIONS array for this specific test case (to make sure they are in fact in the order we want).